lib/semantics/rewrite-parse-tree.cc

```diff
@@ -88,7 +85,7 @@ class RewriteMutator {
 // Check that name has been resolved to a symbol
 void RewriteMutator::Post(parser::Name &name) {
   if (name.symbol == nullptr && errorOnUnresolvedName_) {
-    messages_.Say(name.source, "Internal: no symbol found for '%s'"_en_US,
+    messages_.Say(name.source, "Internal: no symbol found for '%s'"_err_en_US,
```

Comment:
This becomes a `CHECK` in the future, right?

Reply:
Yes, I think after I implement COMMON and EQUIVALENCE I'll change it to a CHECK.